conf/nginx.conf

```diff
@@ -43,8 +43,8 @@ location = /api/v1/videos/upload {
   # Note that temporary space is needed equal to the total size of all concurrent uploads.
   # This data gets stored in /var/lib/nginx by default, so you may want to put this directory
   # on a dedicated filesystem.
-  client_max_body_size                      12G; # default is 1M
-  more_set_headers "X-File-Maximum-Size : 8G always"; # inform backend of the set value in bytes before mime-encoding (x * 1.4 >= client_max_body_size)
+  client_max_body_size                      24G; # default is 1M
+  more_set_headers "X-File-Maximum-Size : 16G always"; # inform backend of the set value in bytes before mime-encoding (x * 1.4 >= client_max_body_size)
```

Comment:
The comment is a bit odd to me.

8*1.4 = 11.2 (that's actually lower than 12, not equal or above to this value)

If there is a mistake in the comment (and should rather be `x * 1.4 <= client_max_body_size`), then we could set the header to `"X-File-Maximum-Size : 17G always"`.

Reply:
I don't know, the file comes from peertube :

https://github.com/Chocobozzz/PeerTube/blob/6c8386bc824522f615a910bd1fdb696f70f1d2a9/support/nginx/peertube#L95-L96